### ISSUE TITLE
YT-VOS 2018 or 2019 ?

### ISSUE THREAD

**Issue by pansanity666** [zlai0/MAST#25]
Hi, 
May I ask which version you used for training? (2018 or 2019)
And did you use the ```train_all_frames.zip``` or ```train.zip``` ?
Thx.

**Comment by wb014** [comment]
the paper mentioned there are 474 videos in validation set, so it's 2018
